### PR TITLE
Allow running the Azure test against the real Azure API in addition to

### PR DESCRIPTION
**Pull request by ppalaga**
local Azurite instance

### CODE REVIEW

**Comment by ppalaga** [comment]
@llowinge you may want to review and test

**Review by oscerd — APPROVED** [review]

File: integration-tests/azure/README.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+== Camel Quarkus Azure (deprecated client v8) integration tests
+
+By default the integration tests run against a local https://github.com/Azure/Azurite[Azurite] container.
+
+To run the tests against the real remote Azure API, you need the follwing:
```
[the remark this answers is not in the document]
Comment (ppalaga) [reply integration-tests/azure/README.adoc:5]: Thanks, let me fix it when we see that the CI is passing.

**Comment by llowinge** [comment]
@ppalaga Thx for the info, i will definitely try it !

**Comment by ppalaga** [comment]
32a8e7c:

* Addressed the typo reported by @jamesnetherton 
* Upgraded to azurite:3.9.0

File: integration-tests/azure/README.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+== Camel Quarkus Azure (deprecated client v8) integration tests
+
+By default the integration tests run against a local https://github.com/Azure/Azurite[Azurite] container.
+
+To run the tests against the real remote Azure API, you need the following:
+
+* An Azure https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-create-account-block-blob?tabs=azure-portal[Storage Account]
```
Comment (llowinge) [inline integration-tests/azure/README.adoc:7]: @ppalaga I'm missing instructions for the Queue Storage.

Comment (ppalaga) [reply integration-tests/azure/README.adoc:7]: There are no specific instructions for queue. The Storage account holds for both. Maybe we should rephrase the readme?

Comment (llowinge) [reply integration-tests/azure/README.adoc:7]: That is weird, i've followed the readme and only the `testAzureBlobComponent` is passing :(

Comment (llowinge) [reply integration-tests/azure/README.adoc:7]: I have it ! One should follow https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=azure-portal as it contains also the Queue service.

Comment (ppalaga) [reply integration-tests/azure/README.adoc]: Thanks, fixed in 58859a2

**Pull request merged**